code/modules/events/infestation.dm

```diff
@@ -1,96 +0,0 @@
-#define LOC_KITCHEN 0
```

Comment:
even when disabling things, don't delete large swathes of code

Reply:
I've removed the entire file, to be honest. this because the removal was requested by West. As described in the commit description.